content/editorials/_posts/2018-03-30-is-alphago-zero-overrated.md

```diff
@@ -16,37 +18,39 @@ Let's start with the coverage about AlphaGo Zero, DeepMind's recent successor to
 
 <figure>
      <iframe width="560" height="315" src="https://www.youtube.com/embed/tXlM99xPQC8" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
-     <figcaption>DeepMind's own explanation of why AlphaGo Zero is so exciting. 
+     <figcaption>DeepMind's own explanation of why AG0 is so exciting. 
 </figcaption>
 </figure>
 
-So, surely DeepMind's demonstration that an AI algorithm can manage to achieve superhuman Go and Chess play purely through self-play is a testament to the usefulness of such techniques for solving the hard problems of AI? Well, yes — it has taken the field decades to get here, and like all those slides in presentations about AI point out the [branching factor](http://theai.wiki/Branching%20Factor) of Go does indeed make it a challenging board game. This is also the first time the same algorithm was used to crack both Chess and Go, and was not specifically tailored for it such as was the case with [Deep Blue](https://en.wikipedia.org/wiki/Deep_Blue_(chess_computer)) (the much heralded machine of IBM that was the first to beat humanity's best at Chess) and the original AlphaGo. Therefore,  AlphaGo Zero is certainly monumental and exciting work (and great PR). 
+So, surely DeepMind's demonstration that an AI algorithm can achieve superhuman Go and Chess play purely through self-play is a testament to the usefulness of such techniques for solving the hard problems of AI? Well, to some extent yes — it has taken the field decades to get here, since the [branching factor](http://theai.wiki/Branching%20Factor) of Go does indeed make it a challenging board game. This is also the first a time the same Deep Learning algorithm was used to crack both Chess and Go[^general], and was not specifically tailored for it such as was the case with [Deep Blue](https://en.wikipedia.org/wiki/Deep_Blue_(chess_computer)) (the much heralded machine of IBM that was the first to beat humanity's best at Chess) and the original AlphaGo. Therefore, AG0 is certainly monumental and exciting work (and great PR). 
 
 <figure>
-      <img src="/content/editorials/images/is-alphago-zero-overrated/history.png" alt="Game History"/>
+      <img src="https://draftin.com:443/images/56473?token=3G6YQX_K8LNCSvkf-fxVAxQMIhG2_8ZOd3uGpjtGKF-myi9JaeEyNf4-cb_r3pynXoHYJr8_n2TPZvaY4OedIjA" alt="Game History"/>
```

Comment:
This is not gonna work, I don't think.

Reply:
It's gonna work, just not as well. I did this for HackMD preview.